snaps/concepts/user-interface.md

```diff
@@ -17,8 +17,8 @@ Other than the settings page, a Snap can modify the MetaMask UI using
 [custom UI](../how-to/use-custom-ui.md) in only two ways:
 
 - By opening a dialog using the [`snap_dialog`](../reference/rpc-api.md#snap_dialog) RPC method.
-- By returning transaction insights from the [`onTransaction`](../reference/exports.md#ontransaction)
-  export.
+- By returning transaction insights from the [`onTransaction`](../reference/entry-points.md#ontransaction)
+  entry point.
```

Comment:
I wonder what is actually the right term to use here. Not export, but should it be entry point or handler?

Reply:
How about "entry point handler"?